test/parallel/test-child-process-kill-numeric.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
+var spawn = require('child_process').spawn;
+
+var exitCode;
+var termSignal;
+var gotStdoutEOF = false;
+var gotStderrEOF = false;
+
+var cat = spawn(common.isWindows ? 'cmd' : 'cat');
+
+
+cat.stdout.on('end', function() {
+  gotStdoutEOF = true;
+});
+
+cat.stderr.on('data', function(chunk) {
+  assert.ok(false);
+});
+
+cat.stderr.on('end', function() {
```

Comment:
Please add `common.mustCall()` around the callback, then gotStderrEOF can be completely removed.